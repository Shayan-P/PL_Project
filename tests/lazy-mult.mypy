def id(a=10):
    if a==0:
        return 0;
    else:  
        print(222222);
        print(a);
        return id(a-1)+1;
        ;
    ;
def meaningless(a=10):
    return 0;
    ;
a=id(6);
b=a+1;
print(id(1)*id(4));
print(id(2)*id(0)*id(8));
