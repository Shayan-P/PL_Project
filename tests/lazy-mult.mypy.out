(num-val 222222)
(num-val 1)
(num-val 222222)
(num-val 4)
(num-val 222222)
(num-val 3)
(num-val 222222)
(num-val 2)
(num-val 222222)
(num-val 1)
(num-val 4)
(num-val 222222)
(num-val 2)
(num-val 222222)
(num-val 1)
(num-val 0)
'finished
